### ISSUE TITLE
Does not recalculate automatically the date / Не пересчитывает автоматически дату

### ISSUE THREAD

**Issue by MVGSoft** [community-ssu/clock-ui#13]
Before up-dating, the system automatically calculated date for an alarm clock (the next day) if it wasn't specified specific value. Now it is necessary to select day, and also call time, that is inconvenient. :disappointed:

До обновления, система автоматически рассчитывала дату для будильника (следующий день), если не было указано конкретного значения. Теперь надо выбрать день, а также время звонка, что неудобно. :disappointed:

**Comment by sixwheeledbeast** [comment]
I don't agree. I believe the new alarm clock is more convenient. It depends on how you use the alarm clock.
I regularly set alarms an hour or two hours ahead throughout the day, in which case I only have to set the time.
The date feature in worldclock provides extra flexibility. Before we could only use the calender for these >24hr ahead alarms.

The only solution I could see is to maybe have a "classic mode" option which defaults to the stock 24 hour only alarm mode?

**Comment by MVGSoft** [comment]
Не знаю... Ранее я просто выбирал время сигнала, нажимал кнопку "Сохранить" и будильник активировался.
Теперь же мне нужно дополнительно активировать флажок "Будильник включен". А в некоторых случаях еще и выставить дату и/или промежуток повтора.

Нужно упрощать интерфейс, либо что-то делать уже активным... Например, автоматически ставить флажок "Будильник включен". Ведь нет смысла только выбрать время, чтобы проснуться, но при этом не включить сам будильник (этим флажком). И что-то сделать с повтором сигнала, который иногда включается сам только по определенным дням, хотя мне этого не требуется.

Да, я бы хотел вернуться к "классическому" режиму, при этом имея возможность назначить другую мелодию. Вместо этого сейчас есть неактивная кнопка (?) с названием мелодии.
Это не вещь первой необходимости, но полезно. Как, например, назначение разных рингтонов разным контактам. =)

**Comment by MVGSoft** [comment]
I don't know... Earlier I just chose signal time, pressed the Keep button and the alarm clock was activated.
Now I need to activate in addition a tag "The alarm clock is included". And in certain cases also to expose date and/or an interval of repetition.

It is necessary to simplify the interface, or to do something already active... For example, automatically to put a tag "The alarm clock is included". There is no sense only to choose time to wake up, but at the same time not to include an alarm clock (this tag). And to make something with repetition of a signal which sometimes joins itself only on certain days though (by default) it isn't required to me.

Yes, I would like to return to the "classical" mode, at the same time having an opportunity to appoint other melody. Instead now there is an inactive button (?) with the name of a melody.
It is not a thing of prime necessity, but it is useful. As, for example, purpose of different ringtones to different contacts. =)
